src/FilenameParsing/NaturalFileIDHelper.php

```diff
@@ -49,6 +51,13 @@ public function lookForVariantIn(ParsedFileID $parsedFileID)
         return $folder == '.' ? '' : $folder;
     }
 
+    public function getVariantGlob(string $folder, ParsedFileID $parsedFileID): string
+    {
+        $fileWithoutFolder = preg_replace('#^' . preg_quote($folder, '#') . '/#', '', $parsedFileID->getFilename());
+        $fileWithoutExtension = preg_replace('#\.' . preg_quote(File::get_file_extension($fileWithoutFolder), '#') . '$#', '', $fileWithoutFolder);
+        return $fileWithoutExtension . FileIDHelper::VARIANT_SEPARATOR . '*';
```

Comment:
```suggestion
        $folderRx = '#^' . preg_quote($folder, '#') . '/#';
        $fileWithoutFolder = preg_replace($folderRx, '', $parsedFileID->getFilename());
        $extRx = '#\.' . preg_quote(File::get_file_extension($fileWithoutFolder), '#') . '$#';
        $fileWithoutExtension = preg_replace($extRx, '', $fileWithoutFolder);
        return $fileWithoutExtension . FileIDHelper::VARIANT_SEPARATOR . '*';
```
Make it a lot easier to read

Reply:
Done